catalog/app/utils/GraphQL/Provider.tsx

```diff
@@ -337,9 +338,13 @@ export default function GraphQLProvider({ children }: React.PropsWithChildren<{}
                   R.evolve({ admin: { user: { list: rmUser } } }),
                 )
               }
-              if (result.admin?.setSsoConfig?.__typename === 'Ok') {
+              if (result.admin?.setSsoConfig?.__typename === 'SsoConfig') {
                 cache.invalidate({ __typename: 'Query' }, 'admin')
                 cache.invalidate({ __typename: 'Query' }, 'roles')
+                cache.updateQuery(
+                  { query: SSO_CONFIG_QUERY },
+                  () => result.admin.setSsoConfig,
```

Comment:
this should have the same structure as the query: `{ admin: { ssoConfig: theActualConfigObject } }`